From c520a4af51d1011644541a53925c77d9addedf32 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Wed, 23 Mar 2016 23:01:38 -0700
Subject: [PATCH] init: Never install Cyanogen recovery

---
 rootdir/init.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/rootdir/init.rc b/rootdir/init.rc
index 4b644bc..baaf352 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -697,8 +697,8 @@ service flash_recovery /system/bin/install-recovery.sh
     disabled
 
 # update recovery if enabled
-on property:persist.sys.recovery_update=true
-    start flash_recovery
+#on property:persist.sys.recovery_update=true
+#    start flash_recovery
 
 service racoon /system/bin/racoon
     class main
